lib/app/ctx.ex

```diff
@@ -227,7 +231,17 @@ defmodule App.Ctx do
       ** (Ecto.NoResultsError)
 
   """
-  def get_person!(id), do: Repo.get!(Person, id)
+  def get_person!(id) do
+    Repo.get!(Person, id)
+    # |> Repo.preload(sessions: :person)
```

Comment:
If we don't need to `preload` the sessions, this function can be restored to `def get_person!(id), do: Repo.get!(Person, id)`
Not urgent.